src/bindings/python/flux/job/Jobspec.py

```diff
@@ -647,6 +647,123 @@ def _v1_validate(resources, tasks, kwargs):
         if not isinstance(system["duration"], numbers.Number):
             raise ValueError("attributes.system.duration must be a number")
 
+    @classmethod
+    # pylint: disable=too-many-branches,too-many-locals,too-many-statements
+    def per_node(
```

Comment:
Maybe per_resource? Since this can do per_{core,node}?

Reply:
Good point!